### ISSUE TITLE
[Request] Screen configuration

### ISSUE THREAD

**Issue by robotgryphon** [rwtema/extrautilities_old_issues#1180]
Just two small requests: One, configure max screen linking from 8x8 to something else, and two, make it configurable to have a max image file size. Yes, the size is workable, but HD is a thing. >.>

**Comment by Waisse** [comment]
Agreed, some improvement would nice : configurable max file size, and make screens render on the full block instead of those black lines on the borders.

**Comment by ben-mkiv** [comment]
would be great to have options for some stuff like:
*max link width/height
*url prefix/suffix
*max filesize
*option to configure GP usage
